android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -144,31 +236,294 @@ private String getCurrentCountry() {
     return current.getCountry();
   }
 
-  private Boolean isEmulator() {
+  private boolean isSupportTelePhony() {
```

Comment:
suggest name change to hasTelephony() (or at least to have the 'P' lower-cased to match other usage, like 'isSupportTelephony')

Reply:
I renamed as you advice...
https://github.com/react-native-community/react-native-device-info/pull/721/commits/72aa24ded9aa61ae5d380f3138b4928d9bbab7d0